MaxMind.MinFraud.UnitTest/Request/TestHelper.cs

```diff
@@ -111,13 +111,32 @@ public static Transaction CreateFullRequest()
                 );
         }
 
-        public static string CurrentDirectory =>
-            Directory.GetCurrentDirectory();
+        private static string GetTestDirectory()
```

Comment:
[minor]

IMO, we can still seize a tiny optimization opportunity here by making use of the static prop the result by doing:

```c#
public static string TestDirectory { get; } = GetTestDirectory();

// the later in this file

File.ReadAllText(Path.Combine(TestDirectory, "TestData",
```

and since it is only used in the same class, make it `private` until we really need to make it public